src/App.test.js

```diff
@@ -1,8 +1,54 @@
 import { render, screen } from '@testing-library/react';
-import App from './App';
+import userEvent from '@testing-library/user-event';
+import App from './app';
```

Comment:
When I tried to run it, it didn't work unless `./App` was capitalized

Reply:
For some reason it works for me with './app' but I made the change to './App'.